### PR TITLE
Set topic namespace in interactive markers display

### LINKED ISSUE #658: Interactive markers when selected through "add" menu by topic doesn't apply topic
An example: if you select interactive markers by a topic publishing it, add it to your rviz window, it doesn't populate the selected topic name like other display types do. You have to go in and then also add the "namespace" that you had previously selected.

### PR DESCRIPTION
**Pull request by rebecca-butler**
Fixes #658 

The interactive markers display inherits from `Display` instead of `RosTopicDisplay`, so it was missing the `setTopic()` function. With these changes, the topic namespace will now be set by default when adding the interactive markers display by topic.

Signed-off-by: Rebecca Butler <rebecca@openrobotics.org>

### CODE REVIEW

File: rviz_default_plugins/src/rviz_default_plugins/displays/interactive_markers/interactive_marker_display.cpp [hunk 1]
```diff
@@ -143,6 +143,17 @@ void InteractiveMarkerDisplay::onDisable()
   unsubscribe();
 }
 
+void InteractiveMarkerDisplay::setTopic(const QString & topic, const QString & datatype)
+{
+  (void) datatype;
+
+  // Separate the namespace from the rest of the topic name, usually formatted '/namespace/update'
+  std::string s = topic.toStdString();
```
Comment (jacobperron) [inline rviz_default_plugins/src/rviz_default_plugins/displays/interactive_markers/interactive_marker_display.cpp]: Avoid single characters for variable names (unless it's a counter in a for-loop). Prefer a more descriptive name like "topic_name".

**Review by jacobperron — COMMENTED** [review]
LGTM, one minor nitpick.

File: rviz_default_plugins/src/rviz_default_plugins/displays/interactive_markers/interactive_marker_display.cpp (hunk 1 above)
Comment (rebecca-butler) [reply rviz_default_plugins/src/rviz_default_plugins/displays/interactive_markers/interactive_marker_display.cpp]: Done in 264cd71

**Comment by rebecca-butler** [comment]
CI build: `--packages-up-to rviz_default_plugins`
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/14897/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/9640/)
* macOS [Build Status](https://ci.ros2.org/job/ci_osx/12577/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/15052/)

**Review by jacobperron — APPROVED** [review]

**Pull request merged**